erpnext/accounts/doctype/sales_invoice/sales_invoice.json

```diff
@@ -1,4875 +1,4875 @@
 {
- "allow_copy": 0,
- "allow_guest_to_view": 0,
- "allow_import": 1,
- "allow_rename": 0,
- "autoname": "naming_series:",
- "beta": 0,
- "creation": "2013-05-24 19:29:05",
- "custom": 0,
- "default_print_format": "",
- "docstatus": 0,
- "doctype": "DocType",
- "document_type": "",
- "editable_grid": 0,
- "engine": "InnoDB",
+ "allow_copy": 0, 
```

Comment:
Because of space, file showing lot of changes, please remove the space so that we can see the actual changes

Reply:
It was autogenerated by frappe. Maybe I'll do it manually